### ISSUE TITLE
[Caption Request] page 974 "Time Sheet Lines Subform"  + page 973 "Time Sheet Card"

### ISSUE THREAD

**Issue by pri-kise** [microsoft/ALAppExtensions#23963]
**🕮 Describe the request**
Could you please add eplicit translations, that these Groups will be translated?

**Additional context**

```
page 974 "Time Sheet Lines Subform"
    actions
    {
        area(Processing)
        {
            group(Line) //Missing Caption isn't translated
            {
                action(Submit)
                {
```

```al
page 973 "Time Sheet Card"
{
    PageType = Document;
    SourceTable = "Time Sheet Header";
    Caption = 'Time Sheet';
    InsertAllowed = false;
    DataCaptionExpression = GetDataCaption();

    layout
    {
        area(Content)
        {
            group(General) //Missing Caption isn't translated
            {
```

**Comment by JesperSchulz** [comment]
Will get addressed with 22.4.

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 11397.
